velocity_controller1:
  type: "velocity_controllers/JointVelocityController"
  joint: 1_wheel
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
velocity_controller2:
  type: "velocity_controllers/JointVelocityController"
  joint: 2_wheel
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
velocity_controller3:
  type: "velocity_controllers/JointVelocityController"
  joint: 3_wheel
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
velocity_controller4:
  type: "velocity_controllers/JointVelocityController"
  joint: 4_wheel
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
stepper_motor_controller:
  type: "position_controllers/JointPositionController"
  joint: base_link1_part1_joint
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
servo_motor_controller1:
  type: "position_controllers/JointPositionController"
  joint: link1_part2_link2_part1_joint
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
servo_motor_controller2:
  type: "position_controllers/JointPositionController"
  joint: link2_part3_link3_part1_joint
  pid:
    p: 1.0
    i: 0.0
    d: 0.0
servo_motor_controller3:
  type: "position_controllers/JointPositionController"
  joint: link3_part2_link4_part1_joint
  pid:
    p: 1.0
    i: 0.0
    d: 0.0